proposals/improved-interpolated-strings.md

```diff
@@ -383,30 +370,34 @@ if the format hole is side-effecting. We could instead evaluate all format holes
 should just be dropping the argument and immediately returning for this version.
 
 _Answer_: We will have conditional evaluation of the holes.
 
 **~~Open~~ Question**
 
-Do we need to dispose of disposable builder types, and wrap calls with try/finally to ensure that Dispose is called? For example, the interpolated string builder in the bcl might have a
+Do we need to dispose of disposable handler types, and wrap calls with try/finally to ensure that Dispose is called? For example, the interpolated string handler in the bcl might have a
 rented array inside it, and if one of the interpolation holes throws an exception during evaluation, that rented array could be leaked if it wasn't disposed.
 
-_Answer_: No. Builders can be assigned to locals (such as `MyBuilder builder = $"{MyCode()};`), and the lifetime of such builders is unclear. Unlike foreach enumerators, where the lifetime
+_Answer_: No. handlers can be assigned to locals (such as `MyHandler handler = $"{MyCode()};`), and the lifetime of such handlers is unclear. Unlike foreach enumerators, where the lifetime
 is obvious and no user-defined local is created for the enumerator.
 
 ## Other considerations
 
-### Allow `string` types to be convertible to builders as well
+### Allow `string` types to be convertible to handlers as well
 
-For type author simplicity, we could consider allowing expressions of type `string` to be implicitly-convertible to _applicable\_interpolated\_string\_builder\_types_. As proposed today,
-authors will likely need to overload on both that builder type and regular `string` types, so their users don't have to understand the difference. This may be an annoying and non-obvious
+For type author simplicity, we could consider allowing expressions of type `string` to be implicitly-convertible to _applicable\_interpolated\_string\_handler\_types_. As proposed today,
+authors will likely need to overload on both that handler type and regular `string` types, so their users don't have to understand the difference. This may be an annoying and non-obvious
 overhead, as a `string` expression can be viewed as an interpolation with `expression.Length` prefilled length and 0 holes to be filled.
 
-This would allow new APIs to only expose a builder, without also having to expose a `string`-accepting overload. However, it won't get around the need for changes to better conversion from
+This would allow new APIs to only expose a handler, without also having to expose a `string`-accepting overload. However, it won't get around the need for changes to better conversion from
 expression, so while it would work it may be unnecessary overhead.
 
+_Answer_:
+
+We think that this could end up being confusing, and there's an easy workaround for custom handler types: add a user-defined conversion from string.
```

Comment:
Sorry if I missed it... was this discussed at LDM? i.e. I should go ahead and add the implicit cast from to string to each handler?

Reply:
The idea of _always_ having a conversion was discussed at LDM. We didn't discuss whether we should recommend that builders include one default, but I do think we should recommend this after some talking with @MadsTorgersen.